bin/reth/src/cli/db_type.rs

```diff
@@ -101,7 +101,7 @@ impl DatabaseInstance {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use reth_primitives::Chain;
+    use alloy_chains::Chain;
```

Comment:
this should still use from pritimitives

Reply:
ig but the db functions are using alloy_chains::Chain, if I use reth_primitives::Chain it will give a mismatch type error

Reply:
it should be the same exact type.